server/routes/register.js

```diff
@@ -0,0 +1,84 @@
+const express = require("express");
+const router = express.Router();
+const verifyRegister = require("../middlewares/verifyRegister");
+const controller = require("../controllers/registerController");
+const auth = require("../middlewares/auth");
+const { token } = require("morgan");
+const hp = require("../utils/hasProperty");
+
+router.get("/",function(req,res){
+    res.status(200).send({ message: "register page displayed successfully!" });
+    //res.render("register");
+});
+
+router.post("/", function(req, res, next) {
+    const message = validateRequest(req.body);
+    if(!hp.hasValidProperty(message))
+        return res.status(400).send(message);
+    
+    let result = verifyRegister.checkEmailExistedInModel(req,res,next);
```

Comment:
this verifyRegister shouldn't be required, you should get an error from the DB if there is an attempt to save a user with an email that already exists

Reply:
ok. I will change this.